prisma/seed.js

```diff
@@ -123,7 +123,9 @@ async function createPost(userId, content) {
   const post = await prisma.post.create({
     data: {
       userId,
-      content
+      content,
+      createdAt: new Date(),
+      updatedAt: new Date()
```

Comment:
no need to add createdAt and updatedAt - these are inserted automatically by Prisma